src/java.xml/share/classes/javax/xml/transform/Transformer.java

```diff
@@ -124,7 +124,7 @@ public abstract void transform(Source xmlSource, Result outputTarget)
      * @param name The name of the parameter, which may begin with a
      * namespace URI in curly braces ({}).
      * @param value The value object.  This can be any valid Java object. It is
-     * up to the processor to provide the proper object coersion or to simply
```

Comment:
That made me pause: some systems have the notion of _type coercion_; but your change looks right.

Reply:
I also had to read this a couple of time, and check the source.